Neos.Flow/Classes/Security/Authentication/AuthenticationProviderManager.php

```diff
@@ -242,21 +242,23 @@ public function logout(): void
      * Signals that the specified token has been successfully authenticated.
      *
      * @param TokenInterface $token The token which has been authenticated
+     * @param AuthenticationProviderInterface $provider The provider that was involved authenticating the token
      * @return void
      * @Flow\Signal
      */
-    protected function emitAuthenticatedToken(TokenInterface $token): void
+    protected function emitAuthenticatedToken(TokenInterface $token, AuthenticationProviderInterface $provider): void
```

Comment:
Will be breaking other implementation if we dont set default to `null` for $provider?

Reply:
But it's protected so only the `AuthenticationProviderManager` can call it!?